### ISSUE TITLE
autoOK bug on new version

### ISSUE THREAD

**Issue by liitfr** [dmtrKovalenko/material-ui-pickers#116]
Hello !
I've updated `material-ui-pickers` just today.

- if I set my DatePicker `autoOk` prop to false => it works, no issue
- if I set my DatePicker `autoOk` prop to true => error at runtime :  

```
material-ui-pickers.cjs.js:2204 Uncaught TypeError: _this.wrapper.togglePicker is not a function
    at DatePickerWrapper.PickerBase._this.togglePicker (material-ui-pickers.cjs.js:2204)
    at DatePickerWrapper.eval (material-ui-pickers.cjs.js:2217)
    at commitCallbacks (react-dom.development.js:6163)
    at commitLifeCycles (react-dom.development.js:8784)
    at commitAllLifeCycles (react-dom.development.js:9946)
    at HTMLUnknownElement.callCallback (react-dom.development.js:542)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:581)
    at invokeGuardedCallback (react-dom.development.js:438)
    at commitRoot (react-dom.development.js:10050)
    at performWorkOnRoot (react-dom.development.js:11017)
```
let me know if you need more details !
Mathias

**Comment by dmtrKovalenko** [comment]
Oh thats looks bad, we need to fix that asap

**Comment by dmtrKovalenko** [comment]
Fixed in 1.0.0-beta.10
